flask_nameko/proxies.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, app=None, connect_on_method_call=True):
 
     def init_app(self, app):
         config = dict()
-        for key, val in app.config.iteritems():
+        for key, val in app.config.items():
```

Comment:
Can we swap this out for the `six.iteritems`? Otherwise in Python 2 this uses a list rather than a generator.